### PR TITLE
Apache Traffic Server - document changes.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket:** https://issues.newrelic.com/browse/NR-149716
**Description:** Updated the integration document changes for ATS.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 1]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+
+Goto the `plugin.config` file.
+  ```shell
+  sudo nano /etc/trafficserver/plugin.config
+  ```
+Add the following line `stats_over_http.so` in your `plugin.config` file.
+
+Make sure your metrics are exposed on the below URLs.
+Metrics of name node: `http://YOUR_IP:8080/_stats`
+
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:49]:
```suggestion
## Install the New Relic Apache agent

Our Apache integrations collect network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).

## Expose your metrics

After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

1. Go to the `plugin.config` file.
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the following line `stats_over_http.so` in your `plugin.config` file:

Make sure your metrics are exposed on the below URLs.
Metrics of name node: `http://YOUR_IP:8080/_stats`

```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 2]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
+
+## Expose your metrics
+After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.
+
+Goto the `plugin.config` file.
+  ```shell
+  sudo nano /etc/trafficserver/plugin.config
+  ```
+Add the following line `stats_over_http.so` in your `plugin.config` file.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:45]: @pkudikyala This is missing a code block - please add it here.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 3]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+
+Make sure your metrics are exposed on the below URLs.
+Metrics of name node: `http://YOUR_IP:8080/_stats`
+
 ## Configure NRI-Flex for Apache Traffic Server [#integrate]
 
 Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:
 
-1. Create a file named `nri-flex-apache-traffic-server-config.yml` in this path:
+1. Creates files with  named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml` and `nri-flex-ats-http-config.yml` in this path:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:54]:
```suggestion
1. Creates files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 4]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
 
   ```shell
   cd /etc/newrelic-infra/integrations.d
   ```
-2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `EVENT_TYPE` and `YOUR_DOMAIN` in the created file named `nri-flex-apache-traffic-server-config.yml`.
+2. Your `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml` and `nri-flex-ats-http-config.yml` file should look like this:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:59]:
```suggestion
2. Your `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` file should look like this:
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 5]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+
   The value on the `event_type` is used to store metrics on the NRDB.
 
-  Example:
-    * `EVENT_TYPE` can be updated to `ATSSampleMetrics`
+  File: `nri-flex-apache-traffic-server-config.yml`
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:63]:
```suggestion
Your `nri-flex-apache-traffic-server-config.yml` file should look like this:
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 6]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+
+## Install Apache New Relic Agent
+Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
+To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
+If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
+
+## Expose your metrics
+After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.
+
+Goto the `plugin.config` file.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:41]:
```suggestion
1. Go to the `plugin.config` file.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 2 above)
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:45]:
```suggestion
After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

Goto the `plugin.config` file.
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the following line `stats_over_http.so` in your `plugin.config` file.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 7]
```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
...
+  ```shell
+  sudo nano /etc/trafficserver/plugin.config
+  ```
+Add the following line `stats_over_http.so` in your `plugin.config` file.
+
+Make sure your metrics are exposed on the below URLs.
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:47]: @pkudikyala What are the below URLs? Are they missing here?

Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:47]:
```suggestion
2. Make sure your metrics are exposed on the below URLs.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 2 above)
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:45]:
```suggestion
2. Add the following line `stats_over_http.so` in your `plugin.config` file.
```

**Review by ally-sassman — CHANGES_REQUESTED** [review]
See my comments

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 2 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the code blocks. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 3 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 4 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:62]: Hi @ally-sassman,
As per your suggestion, I updated the code blocks. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 5 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 6 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 7 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
Here are no more URLs, I updated the description. Kindly review it.

Thanks.

Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx (hunk 2 above)
Comment (pkudikyala) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx]: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 8]
```diff
@@ -29,20 +29,39 @@ You can install the infrastructure agent two different ways:
...
+3. Make sure your metrics are exposed on the below URL.
+
+   Metrics of your Apache Traffic Server: `http://YOUR_IP:8080/_stats`
+
 ## Configure NRI-Flex for Apache Traffic Server [#integrate]
 
 Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:
 
-1. Create a file named `nri-flex-apache-traffic-server-config.yml` in this path:
+1. Creates files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:57]:
```suggestion
1. Create three files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx [hunk 9]
```diff
@@ -29,43 +29,95 @@ You can install the infrastructure agent two different ways:
...
+3. Make sure your metrics are exposed on the below URL.
+
+   Metrics of your Apache Traffic Server: `http://YOUR_IP:8080/_stats`
+
 ## Configure NRI-Flex for Apache Traffic Server [#integrate]
 
 Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:
 
-1. Create a file named `nri-flex-apache-traffic-server-config.yml` in this path:
+1. Go to the `integrations.d` file:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx:57]:
```suggestion
1. Go to the `integrations.d` directory:
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**